### PR TITLE
Add support for PHP 8.2

### PR DESCRIPTION
**Pull request by delboy1978uk**
addresses https://github.com/webfactory/slimdump/issues/103

### CODE REVIEW

**Comment by mpdude** [comment]
The `composer.lock` file is not relevant (and does not block you) when installing this as a Composer dependency in your project. But it helps us to build `.phar` releases with reproducible dependency versions. 

So, I think we should keep it.

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "bin/slimdump"
     ],
     "require": {
-        "php": "^7.2|8.0.*|8.1.*",
+        "php": "^7.2|^8.0",
```
[the remark this answers is not in the document]
Comment (delboy1978uk) [reply composer.json:18]: I guess  we can make it < 8.3 than? tests pass etc on 8.2

[the remark this answers is not in the document]
Comment (mpdude) [reply composer.json:18]: `>= 7.2, < 8.3` should work, right?

**Comment by delboy1978uk** [comment]
ok the lock is reinstated, and  the versions are constrained :-D thanks for the incredibly speedy review!

File: composer.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (delboy1978uk) [reply composer.json]: i have it `^7.2|<8.3`

**Comment by mpdude** [comment]
> thanks for the incredibly speedy review!

To be honest, pure coindicence I happened to work on this project just today.

**Comment by mpdude** [comment]
Seems I need to run a `composer update` to get the lock file in sync; that means I'll upgrade dependencies to the latest versions of packages available for PHP 7.2 as of today.

**Comment by mpdude** [comment]
@delboy1978uk is #107 fine for you as well?

**Comment by delboy1978uk** [comment]
@mpdude great! thanks again :-D

**Comment by mpdude** [comment]
Replaced by #107.

**Pull request closed without merging**